pkgs/applications/virtualization/qemu/default.nix

```diff
@@ -115,8 +115,13 @@ stdenv.mkDerivation rec {
       url = "https://gitlab.com/raboof/qemu/-/commit/3fb5e8fe4434130b1167a995b2a01c077cca2cd5.patch";
       sha256 = "sha256-evzrN3i4ntc/AFG0C0rezQpQbWcnx74nXO+5DLErX8o=";
     })
+    # fix 9p on macOS host, landed in master
```

Comment:
Patch names (using e.g. `name = "foo.patch";`) are generally better than comments.

Reply:
Since other items here don't use `name`, I think it should be fine to include another one without it. But I'll add a name if more changes will be required.